config/fields/mixins/pagepicker.php

```diff
@@ -3,26 +3,42 @@
 return [
     'methods' => [
         'pagepicker' => function (array $params = []) {
-            $query = $params['query'] ?? null;
-            $model = $this->model();
-            $site  = $this->kirby()->site();
-
-            if ($query) {
-                $pages = $model->query($query, 'Kirby\Cms\Pages');
-                $self  = null;
+            $query    = $params['query'] ?? null;
+            $subpages = $params['subpages'] ?? true;
+            $model    = $this->model();
+            $site     = $this->kirby()->site();
+            $root     = $site;
```

Comment:
![FireShot Capture 006 - Sandbox - Mægazine - localhost](https://user-images.githubusercontent.com/3393422/66274257-46585800-e885-11e9-830d-ebe476eda611.png)

This page is root but showing back button.
Therefore keeping `root` page for show/hide back button.